packages/docusaurus-plugin-content-docs/src/__tests__/metadata.test.ts

```diff
@@ -79,7 +79,7 @@ describe('simple site', () => {
 
     expect(data).toEqual({
       id: 'foo/baz',
-      permalink: '/docs/foo/baz',
+      permalink: '/docs/any/pathname.html', // pathname is not "affected" by markdown folder
```

Comment:
if a markdown file is in a subfolder, not sure we want to force the user to have a pathname prefixed by that folder's name. He may group files logically by folder, but still want custom pathnames? don't know